src/index.js

```diff
@@ -79,11 +79,9 @@ class Toggle extends Component {
   setOff = this.setOnState.bind(this, false)
   toggle = this.setOnState.bind(this, undefined)
 
-  componentDidUpdate(prevProps, prevState) {
-    const on = this.getOn()
-
-    if (this.getOn(prevState, prevProps) !== on) {
-      this.props.onToggle(on, this.getTogglerStateAndHelpers())
+  componentWillReceiveProps(newProps) {
```

Comment:
@bslinger @kentcdodds this is sort of a breaking change - `onToggle` was previously called AFTER the component update and this might be important in example in case of DOM reads